staging/src/k8s.io/apiserver/pkg/authorization/authorizerfactory/delegating.go

```diff
@@ -35,12 +36,24 @@ type DelegatingAuthorizerConfig struct {
 	// DenyCacheTTL is the length of time that an unsuccessful authorization response will be cached.
 	// You generally want more responsive, "deny, try again" flows.
 	DenyCacheTTL time.Duration
+
+	// WebhookRetryBackoff specifies the backoff parameters for the authorization webhook retry logic.
+	// This allows us to configure the sleep time at each iteration and the maximum number of retries allowed
+	// before we fail the webhook call in order to limit the fan out that ensues when the system is degraded.
+	WebhookRetryBackoff *wait.Backoff
 }
 
 func (c DelegatingAuthorizerConfig) New() (authorizer.Authorizer, error) {
+	// Provide a default if WebhookRetryBackoff has not been set by the user.
+	retryBackoff := c.WebhookRetryBackoff
```

Comment:
I'd rather fail on this and return a clear error.  This is a coding error and most people should be using our recommended options which should have this value set.

Reply:
@deads2k I agree, I fixed it, we throw clear error now. I originally misinterpreted your comment https://github.com/kubernetes/kubernetes/pull/95705#discussion_r511133119 to add defaulting here.